src/main/java/com/cloudbees/jenkins/plugins/gcloudsdk/GCloudInstallation.java

```diff
@@ -0,0 +1,65 @@
+package com.cloudbees.jenkins.plugins.gcloudsdk;
+
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.EnvironmentSpecific;
+import hudson.model.Node;
+import hudson.model.TaskListener;
+import hudson.slaves.NodeSpecific;
+import hudson.tools.ToolDescriptor;
+import hudson.tools.ToolInstallation;
+import hudson.tools.ToolProperty;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
```

Comment:
If you add a Javadoc header, please describe the class and add <code>@since TODO</code>